src/briefcase/platforms/linux/system.py

```diff
@@ -0,0 +1,1003 @@
+            (rpmbuild_path / "SRPMS").mkdir(parents=True)
+            (rpmbuild_path / "SPECS").mkdir(parents=True)
+
+        # Add runtime package dependencies. App config has been finalized,
+        # so this will be the target-specific definition, if one exists.
+        system_runtime_requires = [
+            "python3",
+        ] + getattr(app, "system_runtime_requires", [])
+
+        # Write the spec file
+        with self.input.wait_bar("Write RPM spec file..."):
+            with (rpmbuild_path / "SPECS" / f"{app.app_name}.spec").open(
+                "w", encoding="utf-8"
+            ) as f:
+                f.write(
+                    "\n".join(
+                        [
+                            # By default, rpmbuild thinks all .py files are executable,
+                            # and if a .py doesn't have a shebang line, it will
+                            # tell you that it will remove the executable bit -
+                            # even if the executable bit isn't set.
+                            # We disable the processor that does this.
+                            "%undefine __brp_mangle_shebangs",
+                            # Base package metadata
+                            f"Name:           {app.app_name}",
+                            f"Version:        {app.version}",
+                            f"Release:        {getattr(app, 'revision', 1)}%{{?dist}}",
+                            f"Summary:        {app.description}",
+                            "",
+                            f"License:        {getattr(app, 'license', 'Unknown')}",
+                            f"URL:            {app.url}",
+                            "Source0:        %{name}-%{version}.tar.gz",
+                            "",
+                        ]
+                        + [
+                            f"Requires:       {requirement}"
+                            for requirement in system_runtime_requires
+                        ]
+                        + [
+                            "",
+                            f"ExclusiveArch:  {self.tools.host_arch}",
+                            "",
+                            "%description",
+                            app.long_description,
+                            "",
+                            # Disable all the auto-detection that tries to magically
+                            # determine requirements from the binaries
+                            f"%global __requires_exclude /usr/lib/{app.app_name}",
+                            f"%global __provides_exclude /usr/lib/{app.app_name}",
+                            # Disable all build post-processing so that binary
+                            # wheels aren't munged.
+                            "%global _enable_debug_package 0",
+                            "%global debug_package %{nil}",
+                            "%global __os_install_post /usr/lib/rpm/brp-compress %{nil}",
```

Comment:
Looking at this more closely at this approach....this will completely replace the post install steps.

On `fedora:37`:
```
[root@146e897129f8 /]# rpm --eval %{__os_install_post}

    /usr/lib/rpm/brp-compress /usr 
    /usr/lib/rpm/brp-elfperms 
    /usr/lib/rpm/brp-strip /usr/bin/strip 
    /usr/lib/rpm/brp-strip-static-archive /usr/bin/strip 
    /usr/lib/rpm/brp-strip-comment-note /usr/bin/strip /usr/bin/objdump 
    /usr/lib/rpm/brp-remove-la-files 
```

I'm not sure we necessarily want to replace all of these steps without cause.

If we just want to avoid stripping, we can instead replace the `strip` binary with an effective noop. For instance:
```
"%define __strip /bin/true"
```

Relevant section of `rpm` build process:
```
Docker| + cp -r usr /app/rpmbuild/BUILDROOT/pysidepkg-0.0.1-1.fc37.x86_64/usr
Docker| + /usr/lib/rpm/check-buildroot
Docker| + /usr/lib/rpm/redhat/brp-ldconfig
Docker| + /usr/lib/rpm/brp-compress
Docker| + /usr/lib/rpm/brp-strip /bin/true
Docker| + /usr/lib/rpm/brp-strip-comment-note /bin/true /usr/bin/objdump
Docker| + /usr/lib/rpm/redhat/brp-strip-lto /bin/true
Docker| + /usr/lib/rpm/brp-strip-static-archive /bin/true
Docker| + /usr/lib/rpm/check-rpaths
Docker| + /usr/lib/rpm/brp-remove-la-files
Docker| + /usr/lib/rpm/redhat/brp-python-bytecompile '' 1 0
Docker| + /usr/lib/rpm/redhat/brp-python-hardlink
```

Reply:
I've taken a slightly different approach here - I found a way to actually disable the strip subcommands, rather than just making the strip call a no-op.

I also needed to disable the rpath check, as it was flagging the .SO files as having an invalid RPATH.